WaterConsumption/examples/example.jsonld

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
this should be name example-normalized.jsonld.
example.jsonld is for key-values format

Reply:
Changed